torch/_inductor/ir.py

```diff
@@ -104,6 +108,25 @@ def reads_from_conv(buf, var_ranges):
     return False, None
 
 
+def ir_node_to_tensor(x, guard_shape=True):
```

Comment:
so for now we still expect fallback kernels to hardcode the shapes? We'd need to address this for dynamic shapes

Reply:
That's what the current state is, yea. But I imagine the change set to enable dynamic shapes here will be pretty small - construct the FakeTensors with the symbolic shapes instead of the real ones